template/docs/index.html

```diff
@@ -87,13 +87,13 @@ <h3>[namespace]-[name]: default</h3>
 
 
     <!-- for initial build and test only -->
-    <script src="./polyfills.js"></script>
-    <script src="./[namespace]-[name]__bundled.js"></script>
+    <script src="/dist/[namespace]-[name]__bundled.js" type="module"></script>
```

Comment:
Since we are doing all of this, we should put this file in a better place. The original reason that I put if in `docs/` was so that the page could be served up as a Github page. Well, I just never followed up with that. 

1. what is a better place? 
1. Should this page be served up when you run the other demo server? 

Not that this is in the scope of this project, but another test I have been wanting to build into the components is a screen capture tool. I used to use backstopJS back in the day, but all the cool kids are telling me that cypress.io is the new hotness. So, does that play into the questions I have?

Reply:
Let's split these out into different issues.  Not sure where it makes sense to place the bundle demo page. 

As for screen capture, is the goal there to capture visual regressions? Didn't @gusnaughton mention he was doing something around that?

Reply:
Yes, visual regression. And yes, I believe that @gusnaughton was talking about Cypress. I am thinking that this would be added to each repo to run visual regression tests.